thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/spec/WoWStdRuleDetectorSpec.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.thirdeye.detection.spec;
+
+import org.apache.pinot.thirdeye.dataframe.util.MetricSlice;
+import org.apache.pinot.thirdeye.detection.Pattern;
+
+
+public class WoWStdRuleDetectorSpec extends AbstractSpec {
```

Comment:
Add some comments to the public class.
Again suggest to use MeanVarianceRuleDetectorSpec to make it more generic.

Reply:
I added some comments and change the name, but not sure what kind of comments should be here. Let me know if it's OK.